ui_testing/testcases/basic_tests/test006_orders.py

```diff
@@ -2871,3 +2871,74 @@ def test084_filter_by_changed_by(self):
             self.assertTrue(key_found)
             # close child table
             self.orders_page.close_child_table(source=results[i])
+
+    def test085_same_testunits_in_different_testplans(self):
+        """
+        Order: Add Same test units in different test plan
+        LIMS-4354
+        """
+        self.test_plan_api = TestPlanAPI() 
+        formated_testunit, formatted_article, formatted_material, material_type_id = self.test_plan_api.create_random_data_for_testplan()
+        testplan1 = self.test_plan_api.create_completed_testplan_random_data(formated_testunit=formated_testunit,
```

Comment:
replace this with :
```
response1, testplan1 = self.create_testplan(testUnits=[formated_testunit],
                                                 selectedArticles=[formatted_article],
                                                 materialType=[formatted_material],
                                                 materialTypeId=[material_type_id])

```
Then assert that:
`response1['message'] == 'operation_success':`